kafka/pykafka/broker.py

```diff
@@ -86,9 +86,10 @@ def fetch_messages(self,
         """
         future = self.handler.request(FetchRequest(
             partition_requests=partition_requests,
-            timeout=10000,
-            min_bytes=1,
+            timeout=timeout,
+            min_bytes=min_bytes,
         ))
+        # XXX - this call returns even with less than min_bytes of messages?
```

Comment:
Not sure about this comment. Are you asking if it's possible or have you seen it happen and we don't handle it correctly?